frontend/wallet/schema.graphql

```diff
@@ -43,50 +13,80 @@ type SyncUpdate {
 
 type SnarkWorker {
   key: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  fee: String! @fake(type: money)
+  fee: String! @fake(type: number)
 }
 
-type SnarkFee {
-  snarkCreator: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  fee: String! @fake(type: money)
+type UserCommand {
+  id: String!
+  isDelegation: Boolean!
+  nonce: Int!
+  date: String! @fake(type: pastDate)
+  from: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"]) 
+  to: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
+  amount: String! @fake(type: number)
+  fee: String! @fake(type: number)
+  memo: String @fake(type:hackerPhrase)
+}
+
+type FeeTransfer {
+  recipient: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
+  fee: String! @fake(type: number)
+}
+
+type CoinbaseTransaction {
+	amount: String! @fake(type: number)
+	feeTransfer: FeeTransfer!
+}
+
+type BlockchainState {
+  date: String! @fake(type: pastDate)
+  snarkedLedgerHash: String!
+  stagedLedgerHash: String!
+  supplyIncrease: String! @fake(type: number)
 }
 
-type SnarkFeeUpdate {
-  fee: SnarkFee
-  consensus: ConsensusState!
+type ProtocolState {
+  previousStateHash: String!
+  blockchainState: BlockchainState!
+}
+
+type Transactions {
+  userCommands: [UserCommand!]!
+  feeTransfer: [FeeTransfer!]!
+  coinbase: CoinbaseTransaction!
 }
 
 type Block {
-  coinbase: String! @fake(type: money)
   creator: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  userCommands: [UserCommand]!
-  snarkFees: [SnarkFee]!
+  numConfirmations: Int!
+  stateHash: String!
+  protocolState: ProtocolState!
+  transactions: Transactions!
 }
 
-type BlockUpdate {
-  block: Block!
-  consensus: ConsensusState!
+type BlockConfirmation {
+  stateHash: String!
+  numConfirmations: Int!
 }
 
 type AnnotatedBalance {
-  total: String! @fake(type: money)
-  unknown: String! @fake(type: money)
+  total: String! @fake(type: number)
```

Comment:
Just to be clear `total` is the currency at the point of consensus finality and `unknown` is the balance at the best tip?

Reply:
If that's the case, that is not how it is currently implemented now.

Reply:
unknown is the amount of the total that is unaccounted for by the known transactions